Tensile/SolutionStructs.py

```diff
@@ -2485,11 +2489,27 @@ def assignDerivedParameters(state):
       state["LdsPadB"] = 0 if state["ProblemType"]["TLUB"] else state["VectorWidth"]
       assert(state["LdsPadB"] >= 0)
 
+    if "MatrixInstM" in state:
```

Comment:
Should this check for MatrixInst equal to non-zero value?  This seem a too-subtle way to check for MatrixInst enabled

Reply:
we are using this approach many places. we should have derived parameter like MatrixInstruction for checks.